README.md

```diff
@@ -1 +1,8 @@
-# data-structures-new
\ No newline at end of file
+# data-structures-new
+## LinkedList:
+    * push(val): Inserts the value at the head of the list
+    * pop(): Pops the first value off the head of the list and returns it. Raises an exception if there are no values to return.
+    * size(): Returns the length of the list.
+    * search(val): Returns the node containing the val in the list if it is present. If it is not present it will return None.
+    * remove(node): Removes the given node from the list. If the node is not in the list is will raise an exception.
+    * display(): Returns a unicode string representing the list as if it were a Pthon tuple literal. Ex: "(5, 'Bravo', 9)"
```

Comment:
len(the_list) returns the size of the list
print(the_list) returns what the display() method returns